core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceConfigSetters.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import java.util.concurrent.Executors;
+
+import com.linecorp.armeria.server.annotation.ExceptionHandlerFunction;
+import com.linecorp.armeria.server.annotation.RequestConverterFunction;
+import com.linecorp.armeria.server.annotation.ResponseConverterFunction;
+
+interface AnnotatedServiceConfigSetters extends ServiceConfigSetters {
```

Comment:
Should we make this public because `AnnotatedServiceConfigSetters` is exposed to Javadoc?
```suggestion
public interface AnnotatedServiceConfigSetters extends ServiceConfigSetters {
```

Reply:
It might be worth adding `@UnstableApi` as well

```suggestion
@UnstableApi
interface AnnotatedServiceConfigSetters extends ServiceConfigSetters {
```

Reply:
@ikhoon Isn't it better not to expose this class as we don't for `ServiceConfigSetters`?

Reply:
>  we don't for ServiceConfigSetters?

It makes sense. Please ignore this comment and let's keep it as is.